packages/core/database/lib/query/helpers/streams/readable.js

```diff
@@ -0,0 +1,175 @@
+'use strict';
+
+const { Readable } = require('stream');
+
+const kQueryDone = Symbol('kOpDone');
+const kPerformingQuery = Symbol('kPerformingQuery');
```

Comment:
Although it might be more clear to change the prefix from `k` to `knex`

Reply:
Why not, it's updated!